### ISSUE TITLE
Duplicated messages in Slack

### ISSUE THREAD

**Issue by augi** [JaSei/check_docker_OOMkiller#3]
We are receiving the same message multiple times (for the same container), e.g.
```
OOM killer APP [11:31] 
Container 346464db323fc4168e1afe5668a55440c9dcb522e8b1bf2cc6b9a8c9a6bfb90b (pinto.srv.int.avast.com/avast/idp/file-extractor:1.0.250) was killed by OOM killer
OOM killer APP [11:38] 
Container 346464db323fc4168e1afe5668a55440c9dcb522e8b1bf2cc6b9a8c9a6bfb90b (pinto.srv.int.avast.com/avast/idp/file-extractor:1.0.250) was killed by OOM killer
```

**Comment by JaSei** [comment]
Should be fixed by #5 - its now correct?

**Comment by augi** [comment]
Yep, it seems to be fixed, thanks! :)
